Priyaa/HTML Basics Remake/Assignment-3.html

```diff
@@ -1,35 +1,40 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>Assignment Regn</title>
-    </head>
-    <body>
-        <h1><b>Register</b></h1>
-        <form id = "registration_form">
+
+<head>
+    <title>Assignment-Registration Form</title>
+</head>
+
+<body>
+    <h1><b>Register</b></h1>
+    <form id="registration_form">
         <b>First name:</b>
         <input type="text" name="firstname" placeholder="First Name" required>
         <b>Last name:</b>
-        <input type="text" name="lastname" placeholder="Last Name">
+        <input type="text" name="lastname" placeholder="Last Name" required>
+        <br><br>
+
+        <b>Gender:</b>
+        <label for="male">Male</label>
+        <input type="radio" name="gender" value="male" required>
+        <label for="female">Female</label>
+        <input type="radio" name="gender" value="female">
+        <label for="others">Others</label>
+        <input type="radio" name="gender" value="others">
```

Comment:
`for` corresponds to `id`. Missing `id` attribute. Refer this: [HTML \<label> for Attribute - w3schools](https://www.w3schools.com/tags/att_label_for.asp)